homeassistant/core.py

```diff
@@ -108,9 +109,10 @@
     from .helpers.entity import StateInfo
 
 
-STAGE_1_SHUTDOWN_TIMEOUT = 100
-STAGE_2_SHUTDOWN_TIMEOUT = 60
-STAGE_3_SHUTDOWN_TIMEOUT = 30
+STOPPING_STAGE_SHUTDOWN_TIMEOUT = 20
+STOP_STAGE_SHUTDOWN_TIMEOUT = 80
+FINAL_WRITE_STAGE_SHUTDOWN_TIMEOUT = 60
+CLOSE_STAGE_SHUTDOWN_TIMEOUT = 30
```

Comment:
I renamed the stages as per [bdraco's request](https://github.com/home-assistant/core/pull/91165/commits/eaf6d024bcb05cb4fecb90af487cda9256b74b57#r1163268933). This renaming, accompanied by the editing of the log messages, has caused my [code coverage to plummet](https://github.com/home-assistant/core/pull/91165#issuecomment-1798656458), leaving me unable to write tests due to lack of Python skills until recently, with aid.

Now you want me to change the stage names back. Is that correct?

Reply:
@emontnemery do you want me to rewrite the whole thing again to rename the events, timeouts etc.?

Reply:
I didn't know @bdraco requested that, I'll check with him.

Also, how can renaming some constants cause code coverage to plummet? Or you mean it plummets for this PR because you change already untested code?

Reply:
> Or you mean it plummets for this PR because you change already untested code?

Exactly